src/pybind/mgr/dashboard/frontend/src/app/ceph/cluster/osd/osd-list/osd-list.component.html

```diff
@@ -85,8 +85,11 @@
 
 <ng-template #osdUsageTpl
              let-row="row">
-  <cd-usage-bar [total]="row.stats.stat_bytes"
-                [used]="row.stats.stat_bytes_used">
+  <cd-usage-bar *ngIf="config$ | async as osdThresholds"
+                [total]="row.stats.stat_bytes"
+                [used]="row.stats.stat_bytes_used"
+                [warningThreshold]="osdThresholds[0].value ? osdThresholds[0].value : osdThresholds[0].default"
+                [errorThreshold]="osdThresholds[1].value ? osdThresholds[1].value : osdThresholds[1].default">
```

Comment:
If value is null/undefined, you can use the [nullish coalescing operator](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing_operator):
```suggestion
                [warningThreshold]="osdThresholds[0].value ?? osdThresholds[0].default"
                [errorThreshold]="osdThresholds[1].value ?? osdThresholds[1].default">
```

Reply:
Thanks @epuertat . Updated the PR